README.md

```diff
@@ -53,12 +53,27 @@ As well, the plates have different culturing conditions, where plate 3 cells wer
 
 ![plate3_nf1_platemap.png](example_figures/plate3_nf1_platemap.png)
 
-It is important to study Schwann cells from NF1 patients because NF1 causes patients to develop neurofibromas, which are red bumps on the skin (tumors) that appear due to the loss of Ras-GAP neurofibromin. 
-This loss occurs when the NF1 gene is mutated (NF1 +/-).
+**Plate 4**
+For plate 4, we will be looking at how using different siRNA constructs to downregulate neurofibromin production in NF1 WT cells impacts the morpholgy as dose increases. 
+We will be able to compare this to controls (e.g., untreated WT and Null cells).
+
+The cells were cultured in 5% FBS.
+
+![plate4_nf1_platemap_genotype.png](example_figures/plate4_nf1_platemap_genotype.png)
+
+There are 8 replicates of NF1 Null cells and the rest of the wells contain NF1 WT cells.
+
+![plate4_nf1_treat_dose.png](example_figures/plate4_nf1_platemap_treat_dose.png)
+
+There are three different siRNA constructs used in this plate, all with the same dose curve from 0.001 nM - 0.1 nM.
+Any well with a 0 nM concentration are not treated with a construct.
 
 ## Goal
 
-The goal of this project is to predict NF1 genotype from Schwann cell morphology. 
+It is important to study Schwann cells from NF1 patients because NF1 causes patients to develop neurofibromas, which are red bumps on the skin (tumors) that appear due to the loss of Ras-GAP neurofibromin. 
```

Comment:
There's a couple things to correct in this. It's probably worth looking up why NF1 deficient Schwann cells are important and the manifestations they cause (it's not red bumps)

Reply:
Sounds good! This statement was created in my first few months, so I would be happy to correct this with my new knowledge!

Reply:
This statement has been corrected.